### DISCUSSION TITLE
VSCode extension to highlight missing translations

### DISCUSSION THREAD

**Discussion by berkayyildiz** [ocombe/ng2-translate#1353] [Show and tell]
I created an extension to highlight missing keys between main and worked on translation, you can check and try here https://github.com/Berkaylar/Find-Missing-Keys

**Comment by rbalet** [comment]
The following vs-code extension seems to be better supported and also handle missing keys https://marketplace.visualstudio.com/items?itemName=Lokalise.i18n-ally .

Closing this topic
